cumulus_library_catalog/catalog/icd10_annotated_counts.py

```diff
@@ -1,61 +1,93 @@
 import cumulus_library
-from cumulus_library import study_manifest
-
+from cumulus_library import errors, study_manifest
 
 
 class IcdCountsBuilder(cumulus_library.CountsBuilder):
     display_text = "Creating ICD10 counts..."
-    def get_icd10_diagnosis_query(self, min_subject: int | None = None):
+
+    def get_icd10_diagnosis_query(self, table_name, depth, min_subject: int | None = None):
         """Convenience function for getting icd10 table
 
         Primarily exists for overriding min_subject for small bins in
         test data.
         """
+        potential_cols = [
```

Comment:
Would this block of logic be easier if you did something like:
```python
potential_cols = [
  [("chapter_code", None), ("chapter_str", None)],
  ...
]
```

Then you could skip the `*2` and `/2` stuff. And adding the cols can look like:
```python
annotation_cols.extend(potential_cols[depth - 1])
```